app/models/mixins/retirement_mixin.rb

```diff
@@ -203,7 +203,7 @@ def retired_event_name
   def raise_retirement_event(event_name, requester = nil)
     requester ||= User.current_user.try(:userid)
     q_options = retire_queue_options
-    $log.info("Raising Retirement Event for [#{name}] with queue options: #{q_options.inspect}")
+    $log.info("Requester [#{requester}] raising Retirement Event for [#{name}] with queue options: #{q_options.inspect}")
```

Comment:
Just wanted to check since `requester` could potentially be `nil` would you want to treat that condition differently or are you ok logging `Requester [] raising Retirement Event for...`

Reply:
@gmcculloug I'm okay with logging ```nil``` for the ```requester```